### PR TITLE
Tweaks package.json and .gitignore

### PR DESCRIPTION
**Pull request by watilde**
##### Summary
+ Lint check as part of `npm test`
+ Add the generated files to .gitignore

See the commit log for more detail.

### CODE REVIEW

**Comment by chriseth** [comment]
Thanks! Github tells me that this branch is out-of-date with the master branch. Could you please rebase it?

**Comment by watilde** [comment]
@chriseth Thanks for the very quick response! I've rebased :)

**Comment by chriseth** [comment]
@axic could you please also take a look here?

File: wrapper.js [hunk 1]
```diff
@@ -207,7 +207,7 @@ function setupMethods (soljson) {
       var url = 'https://ethereum.github.io/solc-bin/bin/soljson-' + versionString + '.js';
       https.get(url, function (response) {
         if (response.statusCode !== 200) {
-          cb('Error retrieving binary: ' + response.statusMessage);
+          cb(new Error('Error retrieving binary: ' + response.statusMessage));
```
Comment (axic) [inline wrapper.js]: Cool, the new semistandard can catch this?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "lint": "semistandard",
     "prepublish": "./downloadCurrentVersion.js && ./verifyVersion.js",
+    "pretest": "npm run lint",
```
Comment (axic) [inline package.json]: Didn't know this existed. CI was running it manually before the test.

Comment (axic) [reply package.json]: For anyone checking, here's the complete list: https://docs.npmjs.com/misc/scripts

**Comment by axic** [comment]
Thanks, everything looks good.

File: wrapper.js (hunk 1 above)
Comment (watilde) [reply wrapper.js]: Yes, it can! When I bumped the version of semistandard, it said: `Unexpected literal in error position of callback.` so I fixed that.

**Comment by watilde** [comment]
Thanks! Cheers🍻

**Pull request merged**